internal/pkg/deploy/cloudformation/cloudformation.go

```diff
@@ -427,3 +428,31 @@ func stopSpinner(spinner *progress.Spinner, err error, label string) {
 	}
 	spinner.Stop(log.Serrorf("%s\n", label))
 }
+
+// upgradeStackUpdateErrorHandling is used to handle the error returned by CFN
+// stack update when upgrading the application/environment. It returns true if the CFN stack
+// is being updated.
+func upgradeStackUpdateErrorHandling(name string, err error) (bool, error) {
```

Comment:
What do you think of:
```suggestion
func isRetryableUpdateError(err error) (bool, error) {
```
And let the client of this fn do the wrapping?